Makefile

```diff
@@ -110,6 +110,7 @@ ISTIO_DOCKER_TAR:=${ISTIO_OUT}/docker
 
 GO_VERSION_REQUIRED:=1.9
 
+# TODO: replace this with DOCKER_ISTIO_HUB (20 lines above)
```

Comment:
DOCKER_ISTIO_HUB => ISTIO_DOCKER_HUB

I added ISTIO_DOCKER_HUB recently when it became known that istioctl was hard-coding a reference to docker.io.  It was late in a release system and I opted for a more specific option for customization.  After some reflection and greater understanding I'm inclined to think that istioctl should not use a separate option name for customizing the hub.  I had in mind to change ISTIO_DOCKER_HUB to HUB (which is mostly just updating this file, release/cloud_builder.sh, and bin/get_workspace_status).  While I like the idea of using a more specific name than just "HUB" or "TAG" (to reduce conflict with other potential uses), I'm afraid there's be a lot of build, test, and dev scripts to update.